packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -168,6 +168,16 @@ let didWarnSetStateChildContext;
 let warnAboutUpdateOnUnmounted;
 let warnAboutInvalidUpdates;
 
+if (enableSchedulerTracking) {
+  // Provide explicit error message when production+profiling bundle of e.g. react-dom
+  // Is used with production (non-profiling) bundle of schedule/tracking
+  invariant(
+    __interactionsRef != null && __interactionsRef.current != null,
+    "Profiling renderer (e.g. 'react-dom/profiling') cannot be used with non-profiling 'schedule/tracking'. " +
+      "To fix, alias 'schedule/tracking' to 'schedule/tracking-profiling'.",
```

Comment:
This would be bundler specific and I’m not sure they all use the “alias” wording.

Another thing to consider is that the person seeing this might be unaware that schedule package exists or what it is.

How about 

```
It is not supported to run the profiling version of a renderer (for example, `react-dom/profiling`) without also replacing the `schedule/tracking` module with `schedule/tracking-profiling`. Your bundler might have a setting for aliasing both modules. Learn more at http://fb.me/react-profiling
```

The link points to a gist explaining how to do it in webpack. Later we change it to point to a blog post.

Reply:
I dig it. 👍 Thanks for the ideas Dan.